pkg/skaffold/deploy/helm/deploy.go

```diff
@@ -256,9 +256,13 @@ func (h *Deployer) Render(ctx context.Context, out io.Writer, builds []graph.Art
 	renderedManifests := new(bytes.Buffer)
 
 	for _, r := range h.Releases {
-		args := []string{"template", chartSource(r)}
+		releaseName, err := util.ExpandEnvTemplateOrFail(r.Name, nil)
```

Comment:
Is `releaseName` the only field that needs this change?

Wasn't sure if these fields referenced in our-docs/the-bug might need changes as well:
deploy.helm.releases.setValueTemplates (see Deploying with helm)
deploy.helm.releases.name (see Deploying with helm) 
deploy.helm.releases.namespace (see Deploying with helm)

Reply:
That is an excellent question — I looked for other uses of `r.Name`, but should have checked the others.

- releases.namespace is expanded in `releaseNamespace()`
- setValueTemplates is expanded in `constructOverrideArgs()`

So I think we're good.